spec/models/invoice_spec.rb

```diff
@@ -60,5 +60,16 @@
         expect(invoice1.potential_revenue(merchant1)).to eq(35000)
       end
     end
+
```

Comment:
we should add more test data to check that it doesn't pick up invoice_items that don't have successful transactions. Per User Story 28 potential_revenue description.

Reply:
Yo, I can definitely work on that